app/app.go

```diff
@@ -350,9 +350,15 @@ func NewApp(
 		}, app.akashAppModules()...)...,
 	)
 
+	// During begin block slashing happens after distr.BeginBlocker so that
+	// there is nothing left over in the validator fee pool, so as to keep the
+	// CanWithdrawInvariant invariant.
+	// NOTE: staking module is required if HistoricalEntries param > 0
+	// NOTE: capability module's beginblocker must come before any modules using capabilities (e.g. IBC)
```

Comment:
Did you find this information somewhere in the Cosmos docs or just figure it out from reading the code?

Reply:
Took it from [cosmos sdk code](https://github.com/cosmos/cosmos-sdk/blob/1dba6735739e9b4556267339f0b67eaec9c609ef/simapp/app.go#L325-L329)

Reply:
Can you put a link to that in the source code? Someone else may want to look at that as well.